BitmovinConvivaAnalytics/Classes/ContentMetadataBuilder.swift

```diff
@@ -69,12 +69,16 @@ class ContentMetadataBuilder {
             if let type = streamType {
                 contentMetadata.streamType = type
             }
-            if let duration = self.duration {
+            if let duration = self.duration, self.duration ?? 0 > 0 {
```

Comment:
You can use the unwrapped value `duration` in follow up conditions 🙂 

```suggestion
            if let duration = self.duration, duration > 0 {
```

Reply:
Much better, thanks!